src/Normalizer.php

```diff
@@ -2,8 +2,6 @@
 
 namespace Aeviiq\ValueObject;
 
-use function Safe\preg_replace;
```

Comment:
Just removing the usage of the safe version does technically change the behavior of the class. So the only question would be if we want to reproduce the behaviour of the safe version. Or just keep it as is. (Causing a slight behavior change)

Reply:
@aeviiq